lib/templates/accountTemplate.js

```diff
@@ -86,23 +95,59 @@ class AccountTemplate {
     fsUtils.writeConfig(this.TEMPLATE_TYPE, name, templateConfig);
   }
 
-  static #prepareConfigDetails(template, existingConfig = {}) {
+  static #prepareConfigDetails(type, envId, template, existingConfig = {}) {
     const attributes = this.CONFIG_ITEMS.reduce((acc, attribute) => {
       if (template.hasOwnProperty(attribute)) {
         acc[attribute] = template[attribute];
       } else if (existingConfig?.hasOwnProperty(attribute)) {
         acc[attribute] = existingConfig[attribute];
       }
+
       return acc;
     }, {});
+
+    // We need to also create the mapping_list_ranges, which could be different per firm & partner
+    const existingMappingRanges = existingConfig.mapping_list_ranges || [];
```

Comment:
It'd be good to have a comment with a sample format of `mapping_list_ranges`. It makes reading this logic easier.

Reply:
Added example of the data in comment

Reply:
Added example of the data in comment

Reply:
Do you mean the one in the end: https://github.com/silverfin/silverfin-cli/blob/6120acb6b736db1d3fc65ca28ef43ba9f5dc82f7/lib/templates/accountTemplate.js#L191 ? There the mapping_list_ranges are empty as far as I can see :thinking:

Reply:
Ahh I was looking at the wrong version, my bad. Resolving it